src/graph/graph.c

```diff
@@ -1518,11 +1562,21 @@ GrB_Matrix Graph_GetRelationMatrix(const Graph *g, int relation_idx) {
 }
 
 // Returns true if relationship matrix 'r' contains multi-edge entries, false otherwise
-bool Graph_RelationshipContainsMultiEdge(const Graph *g, int r) {
+bool Graph_RelationshipContainsMultiEdge
+(
+	const Graph *g,
+	int r,
+	bool transpose
```

Comment:
You're right, reason for differentiating is to avoid flushing both, assume query only needs `Rt` but this check being performed on `R` as such the both matrices are flushed